Using-Mastodon/2FA.md

```diff
@@ -1,8 +1,10 @@
-# 2-Factor Authentication
+# Two Factor Authentication
```

Comment:
Perhaps we should here add "2FA" to ease the search on this well used abbreviation?